apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -13,22 +15,22 @@ using namespace Halide::ConciseCasts;
 Var x, y, c, yi, yo, yii, xi;
 
 // Average two positive values rounding up
-Expr avg(Expr a, Expr b) {
```

Comment:
I think the changes here are a net readability loss for performance penny-pinching. Happy to do this inside the compiler, but it's distracting in code intended to demonstrate how Halide works.